src/transform/CollapseTable.css

```diff
@@ -80,7 +80,7 @@ table.infobox caption {
 }
 
 /* The outermost table container. todo: rename. */
-.app_table_container {
+div.app_table_container {
```

Comment:
This is fallout from the theme-ing 'baseline' approach. Had to use more aggressive CSS to robustly apply themes to un-classed elements with inline styles. Transforms must match or exceed the theme level of selector specificity to be able to override the 'baseline' styles set by the theme transform.